third_party/airflow/examples/hello_armada_auth.py

```diff
@@ -0,0 +1,146 @@
+                        "cpu": api_resource.Quantity(string="120m"),
+                        "memory": api_resource.Quantity(string="510Mi"),
+                    },
+                    limits={
+                        "cpu": api_resource.Quantity(string="120m"),
+                        "memory": api_resource.Quantity(string="510Mi"),
+                    },
+                ),
+            )
+        ],
+    )
+
+    return [
+        submit_pb2.JobSubmitRequestItem(
+            priority=1,
+            pod_spec=pod,
+            namespace="personal-anonymous",
+            annotations={"armadaproject.io/hello": "world"},
+        )
+    ]
+
+
+def creds_callback():
+    return grpc.composite_channel_credentials(
+        channel_credentials,
+        grpc.metadata_call_credentials(
+            GrpcBasicAuth(username="armada-user", password="armada-pass")
+        ),
+    )
+
+
+"""
+This is an example of a Airflow dag that
+uses a BashOperator and an ArmadaOperator
+"""
+with DAG(
+    dag_id="hello_armada_auth",
+    start_date=pendulum.datetime(2016, 1, 1, tz="UTC"),
+    schedule_interval="@daily",
+    catchup=False,
+    default_args={"retries": 2},
+) as dag:
+    """
+    The ArmadaOperator requires grpc.channel arguments for armada and
+    the jobservice.
+    """
+    channel_credentials = grpc.local_channel_credentials()
+
+    armada_channel_args = {
+        "target": "localhost:50051",
+        "credentials_callback_args": {
+            "module_name": "hello_armada_auth",
+            "function_name": "creds_callback",
+            "function_kwargs": {},
+        },
+    }
+    job_service_channel_args = {"target": "localhost:60003"}
+
+    """
+    This defines an Airflow task that runs Hello World and it gives the airflow
+    task name of dummy.
+    """
+    op = BashOperator(task_id="dummy", bash_command="echo Hello World!")
+    """
+    This is creating an Armada task with the task_id of armada and name of armada.
+    The Airflow operator needs queue and job-set for Armada
+    You also specify the PythonClient and JobServiceClient for each task.
+    You should reuse them for all your tasks.
+    This job will use the podspec defined above.
+    """
+    armada = ArmadaOperator(
+        task_id="armada",
+        name="armada",
+        armada_queue="test",
+        job_service_channel_args=job_service_channel_args,
+        armada_channel_args=armada_channel_args,
+        job_request_items=submit_sleep_job(),
+        lookout_url_template="http://127.0.0.1:8089/jobs?job_id=<job_id>",
+    )
+    """
+    Airflow dag syntax for running op and then armada.
+    """
+
+    op >> armada
+
+
+if __name__ == "__main__":
+    from sqlalchemy import create_engine
+    from sqlalchemy.orm.session import Session
+
+    engine = create_engine("postgresql://airflow:airflow@172.20.0.3:5432/airflow")
+    session = Session(engine)
+
+    # Will need to run `airflow db init` first
+    dag.test(
```

Comment:
Can I ask why you added this?

Reply:
I probably shouldn't leave this in, but it wouldn't connect to the airflow postgres instance without it

Reply:
I managed to get it to run by running `airflow db init` locally. I think this would be an easy win for more CI tests?

Reply:
`airflow db init` shouldn't be necessary after `mage airflow start`